metrics/engine/ingester.go

```diff
@@ -4,10 +4,14 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	"go.k6.io/k6/metrics"
 	"go.k6.io/k6/output"
 )
 
-const collectRate = 50 * time.Millisecond
+const (
+	collectRate          = 50 * time.Millisecond
+	timeSeriesFirstLimit = 100e3 // 100k
```

Comment:
```suggestion
	timeSeriesFirstLimit = 100_000
```

this is a bit more readable